### PR TITLE
lbsd-291 - editor styling

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

**Comment by nistormihai** [comment]
:thumbsup:

**Comment by vladnicoara** [comment]
:+1:

File: app/scripts/ng-sir-trevor-blocks.js [hunk 1]
```diff
@@ -76,9 +76,8 @@ define([
             // NOTE: responsive layout is currently disabled. so row and col-md-6 are useless
                 html: [
                     '<div class="row st-block__upload-container">',
-                    '    <input type="file" type="st-file-upload" />',
                     '    <div class="col-md-6">',
-                    '        <button class="btn btn-default"><%= i18n.t("general:upload") %></button>',
+                    '           <input type="file" type="st-file-upload" />',
```
Comment (vied12) [inline app/scripts/ng-sir-trevor-blocks.js]: It's not possible to add picture by choosing them from the browser right now. It is only by drag and drop. Is it wanted ?

Comment (darconny) [inline app/scripts/ng-sir-trevor-blocks.js]: I just arranged the code for styling issues, browsing input type file was already there.

Comment (vied12) [inline app/scripts/ng-sir-trevor-blocks.js]: But you removed the button `select from folder` to add a file from the browser.
So we can add file only by drag and drop now.

Comment (darconny) [inline app/scripts/ng-sir-trevor-blocks.js]: No its not removed, it is just styled differently.. all code is in <input type="file"> and browsing is working as it should.

Comment (darconny) [inline app/scripts/ng-sir-trevor-blocks.js]: Look at liveblog-edit.less line 233-261, that's the input code that behaves like a default button.

**Comment by vied12** [comment]
:+1:  Works fine now !

**Pull request merged**